### PR TITLE
fix: 933161 regex

### PR DESCRIPTION
**Pull request by theMiddleBlue**
same fix made on #2301

### CODE REVIEW

**Comment by theMiddleBlue** [comment]
converted to draft because of https://github.com/coreruleset/coreruleset/pull/2301#issuecomment-965315843

File: util/regexp-assemble/data/933161.data [hunk 1]
```diff
@@ -24,7 +24,7 @@
 
 ##!+i
 ##!^\b
-##!$(?:\s|/\*.*\*/|//.*|#.*)*\(.*\)
+##!$\(.*\)
```
Comment (theseion) [inline util/regexp-assemble/data/933161.data:27]: After removing whitespace and comment markers you might miss this:
```
=abs\n#dummy comment\n('gotcha')
```
which would be converted to:
```
=absdummycomment('gotcha')
```
Removing whitespace could be ok I guess but removing comment markers will break the rule.

**Comment by dune73** [comment]
Do we have a status here?

**Comment by theMiddleBlue** [comment]
after rebase the regex still vulnerable to redos.
I've just inverted the position of comment sequences and crs-toolchain done the magic!

Now seems fixed:
```shell
Welcome to Regexploit. Enter your regexes:
(?i)\b(?:a(?:bs|cosh?|r(?:ray|sort)|s(?:inh?|(?:o|se)rt)|tan[2h]?)|b(?:asename|indec)|c(?:eil|h(?:dir|eckdate|mod|o(?:p|wn)|root)|lose(?:dir|log)|o(?:(?:mpac|(?:nsta|u)n)t|py|sh?)|(?:ryp|urren)t)|d(?:ate|e(?:coct|fined?)|i(?:(?:skfreespac)?e|r(?:name)?)|(?:oubleva)?l)|e(?:a(?:ch|ster_da(?:te|ys))|cho|mpty|nd|r(?:egi?|ror_log)|x(?:(?:i|trac)t|p(?:lode)?))|f(?:close|eof|gets|ile(?:owner|pro|(?:siz|typ)e)|l(?:o(?:atval|ck|or)|ush)|(?:mo|rea)d|stat|t(?:ell|ok)|unction)|g(?:et(?:date|t(?:ext|ype))|mdate)|h(?:ash|e(?:ader(?:s_(?:lis|sen)t)?|brev)|ypot)|i(?:conv|(?:dat|mplod)e|n(?:(?:clud|vok)e|t(?:div|val))|s(?:_(?:a(?:rray)?|bool|(?:calla|dou)ble|f(?:inite|loat)|in(?:finite|t(?:eger)?)|l(?:ink|ong)|n(?:an|u(?:ll|meric))|object|re(?:al|source)|s(?:calar|tring))|set))|join|k(?:ey|sort)|l(?:(?:cfirs|sta)t|evenshtein|i(?:nk(?:info)?|st)|o(?:caltime|g(?:1[0p])?)|trim)|m(?:a(?:i[ln]|x)|b(?:ereg|split)|etaphone|hash|i(?:crotime|n)|y?sql)|n(?:atsor|ex)t|o(?:ctdec|penlog|rd)|p(?:a(?:ck|thinfo)|close|i|o[sw]|r(?:ev|intf?))|quotemeta|r(?:an(?:d|ge)|e(?:adlin[ek]|(?:cod|nam|quir)e|set|wind)|ound|sort|trim)|s(?:(?:candi|ubst)r|(?:e(?:rializ|ttyp)|huffl)e|i(?:milar_text|nh?|zeof)|leep|o(?:rt|undex)|p(?:liti?|rintf)|qrt|rand|t(?:at|r(?:coll|(?:le|sp)n))|y(?:mlink|slog))|t(?:a(?:int|nh?)|e(?:mpnam|xtdomain)|ime|ouch|rim)|u(?:cfirst|mask|n(?:iqid|link|(?:se|tain)t)|s(?:leep|ort))|virtual|wordwrap)(?:[\s\v]|/\*.*\*/|(?:#|//).*)*\(.*\)
No ReDoS found.
```

rolled back to `t:none` tfunc!
if you agree, we can merge.

**Comment by theseion** [comment]
I don't believe that there's a significant difference between the two versions. I rather think that `regexploit` doesn't know how to find a ReDOS for your new version. `regex101` also reports a very low number of steps for both versions (using PHP and PCRE2), regardless of the number of `#` (I tried with 7000+). TBH, I'm not sure that there really is a ReDOS here (the number of steps does increase depending on the number of characters _after_ the comment character, but that increase is close to linear).

**Comment by dune73** [comment]
Anybody has a status or a plan what to do here?

**Comment by theMiddleBlue** [comment]
I'm all for merging this as is, as regexploit doesn't flag any redos problems anymore.

**Review by dune73 — COMMENTED** [review]
Looks good. We no longer think this is a ReDoS, so let's merge.

**Review by dune73 — APPROVED** [review]

**Pull request merged**